tests/testthat/test-compiler-pt-to-lines.r

```diff
@@ -0,0 +1,14 @@
+acontext("pt/points to lines conversion test")
+
+viz <- list(
+  p1 = ggplot()+
+    geom_point(aes(Petal.Length, Sepal.Length, color=Species),data=iris))
+
+theme.pars <- plot_theme(viz)
+panel_margin_lines <- pt.to.lines(theme.pars$panel.margin)
+
+if(grid::unitType(theme.pars$panel.margin) == "pt" || grid::unitType(theme.pars$panel.margin) == "points") {
+  test_that("after conversion values unequal", {
```

Comment:
maybe add a comment to explain why we want to values to be unequal after conversion? not easy to understand the context of where this test comes from ...

Reply:
Sure thing!